src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7589,6 +7589,7 @@ public sealed partial class StringBuilder : System.Runtime.Serialization.ISerial
         public System.Text.StringBuilder Append(long value) { throw null; }
         public System.Text.StringBuilder Append(object value) { throw null; }
         public System.Text.StringBuilder Append(System.ReadOnlySpan<char> value) { throw null; }
+        public System.Text.StringBuilder Append(System.ReadOnlyMemory<char> value) { throw null; }
```

Comment:
Nit, sorts above line above. Not worth resetting the PR

Reply:
Since i'm still on the branch, fixed.